discovery/discovery.go

```diff
@@ -207,6 +214,25 @@ func (o *orchestratorPool) GetOrchestrators(ctx context.Context, numOrchestrator
 	return ods, nil
 }
 
+func sendLiveAICapacityMetrics(info *net.OrchestratorInfo) {
+	caps := info.Capabilities
+	if !monitor.Enabled {
+		return
+	}
+	if caps == nil || caps.Capacities == nil || caps.Constraints.PerCapability == nil {
+		return
+	}
+	liveAI, ok := caps.Constraints.PerCapability[uint32(core.Capability_LiveVideoToVideo)]
+	if !ok {
+		return
+	}
+
+	for modelID, model := range liveAI.Models {
+		monitor.AIContainersInUse(int(model.CapacityInUse), modelID, info.GetTranscoder())
```

Comment:
I think we need to update those metrics also when there is an error returned from `GetOrchInfo()`. Otherwise, if the Orchestrator is Capped or just stopped working, we'll still be reporting it with the metric `1`.

Reply:
ah yep of course thanks